### PR TITLE
Added "power outlets in the room" feature

### PR DESCRIPTION
**Pull request by frbuccoliero**
I added this feature that displays a little plug emoji next to the rooms that have power outlets. My first approach was to make a request for each room and check in the html of the expanded view of the single room, but this approach ended up slowing the bot too much. This solution instead adds a little json file with the "id_aula" numbers of the rooms that have power outlets, conveniently generated by a little script I added. As I wrote there since we don't change power outlets every day you can just run this script every once and a while and will be fine. 

- Thx to @lorenzocavallero for the idea

### CODE REVIEW

File: search/find_classrooms.py [hunk 1]
```diff
@@ -44,7 +44,11 @@ def find_classrooms(location , day , month , year):
     soup = BeautifulSoup(r.text, 'html.parser')
     tableContainer = soup.find("div", {"id": "tableContainer"})
     tableRows = tableContainer.find_all('tr')[3:] #remove first three headers
-    
+
+    with open("json/roomsWithPower.json","r") as j:
+        rwp = json.load(j)
+        j.close()
```
Comment (feDann) [inline search/find_classrooms.py:50]: redundant

File: search/find_classrooms.py [hunk 2]
```diff
@@ -64,11 +68,13 @@ def find_classrooms(location , day , month , year):
                 if ROOM in td.attrs['class']:
                     room = td.find('a').string.replace(" ","")
                     link = td.find('a')['href']
+                    id_aula = int(link.split("=")[-1])
                     
                     if room not in info[buildingName]:
                         info[buildingName][room] = {}
                         info[buildingName][room]['link'] = BASE_URL + link
                         info[buildingName][room]['lessons'] = []
+                        info[buildingName][room]['powerPlugs'] = id_aula in rwp
```
Comment (feDann) [inline search/find_classrooms.py:77]: should use a set instead of a list for performance

**Review by feDann — CHANGES_REQUESTED** [review]
can you upload a screenshot of the final result?

**Comment by frbuccoliero** [comment]
Screenshot of the new feature as requested:

image

**Pull request merged**